model.py

```diff
@@ -0,0 +1,176 @@
+from config import Configuration
+import logging
+import warnings
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    String
+)
+from sqlalchemy import (
+    create_engine
+)
+from sqlalchemy.exc import (
+    IntegrityError
+)
+from sqlalchemy.ext.declarative import (
+    declarative_base
+)
+from sqlalchemy.orm import (
+    backref,
+    relationship,
+    sessionmaker,
+)
+from sqlalchemy.orm.exc import (
+    NoResultFound,
+    MultipleResultsFound,
+)
+
+from geoalchemy2 import Geography
+
+def production_session():
+    url = Configuration.database_url()
+    if url.startswith('"'):
+        url = url[1:]
```

Comment:
It was a workaround for a bug, like, two years ago. I've taken the code out of here and everything still seems to work.